$OpenBSD: patch-kdoctools_CMakeLists_txt,v 1.2 2013/12/08 19:13:32 zhuk Exp $
Looks like unused in build process for KDE4, so rename it to de-conflict with KDE3.
--- kdoctools/CMakeLists.txt.orig	Wed Aug 28 20:57:06 2013
+++ kdoctools/CMakeLists.txt	Tue Nov  5 11:05:21 2013
@@ -183,7 +183,8 @@ endforeach( _kdeincludexslt ${_kdeincludexslt_ALL} )
 
 install( FILES help.protocol ghelp.protocol  DESTINATION  ${SERVICES_INSTALL_DIR} )
 
-install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/checkXML DESTINATION ${BIN_INSTALL_DIR} )
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/checkXML DESTINATION ${BIN_INSTALL_DIR}
+        RENAME checkXML4 )
 
 if(NOT WIN32)
   install( FILES kio_help.upd  DESTINATION  ${KCONF_UPDATE_INSTALL_DIR} )
